app/Commands/LocalDocker/Share.php

```diff
@@ -0,0 +1,131 @@
+<?php declare( strict_types=1 );
+
+namespace App\Commands\LocalDocker;
+
+use Filebase\Database;
+use App\Contracts\File;
+use App\Contracts\Runner;
+use App\Services\Docker\Local\Config;
+use Illuminate\Filesystem\Filesystem;
+
+/**
+ * Share your local environment using ngrok
+ *
+ * @package App\Commands\LocalDocker
+ */
+class Share extends BaseLocalDocker {
+
+    public const MU_PLUGIN = '0-so-ngrok.local.php';
+
+    /**
+     * The signature of the command.
+     *
+     * @var string
+     */
+    protected $signature = 'share';
+
+    /**
+     * The description of the command.
+     *
+     * @var string
+     */
+    protected $description = 'Share your local project on a temporary URL using ngrok';
+
+    /**
+     * The user's settings database.
+     *
+     * @var \Filebase\Database
+     */
+    protected $settings;
+
+    /**
+     * Share constructor.
+     *
+     * @param  \Filebase\Database  $settings
+     */
+    public function __construct( Database $settings ) {
+        parent::__construct();
+        $this->settings = $settings;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @param  \App\Services\Docker\Local\Config  $config
+     * @param  \App\Contracts\Runner              $runner
+     * @param  \Illuminate\Filesystem\Filesystem  $filesystem
+     * @param  \App\Contracts\File                $file
+     *
+     * @return int
+     * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
+     */
+    public function handle( Config $config, Runner $runner, Filesystem $filesystem, File $file ): int {
+        $settings = $this->settings->get( 'user_secrets' );
+
+        if ( empty( $settings->ngrok_token ) ) {
+            $this->info( 'Ngrok requires a free user account to proxy to https domains. Sign up: https://dashboard.ngrok.com/signup' );
+            $authToken = $this->secret( 'Enter your authtoken found in your dashboard: https://dashboard.ngrok.com/auth/your-authtoken (input hidden)' );
+
+            if ( empty( $authToken ) ) {
+                $this->error( 'No token entered' );
+
+                return self::EXIT_ERROR;
+            }
+
+            $settings->ngrok_token = $authToken;
+            $settings->save();
+        }
+
+        $this->checkGitIgnore( $file, $config->getProjectRoot() );
+
+        $source  = storage_path( sprintf( 'wordpress/mu-plugins/%s', self::MU_PLUGIN ) );
+        $content = $filesystem->get( $source );
+        $target  = sprintf( '%s/%s', $config->getProjectRoot(), sprintf( 'wp-content/mu-plugins/%s', self::MU_PLUGIN ) );
```

Comment:
Shouldn't affect many projects, but any with a different WP_CONTENT_DIR will have issues with this. Maybe doesn't matter. 🤷

Reply:
Good eye! I'll note for the future to add a configuration item to squareone.yml to configure this on a per project basis.